en/docs/usecases/datamapper/read-csv-file-and-transform-to-xml-file.md

```
@@ -0,0 +1,618 @@
+
+Inside the Data Mapper view, you’ll see two main panels:
+
+#### Left Panel — Input Data
+- `csvRecords` → Type: `CSV[]`
+- `inputCSV` → Type: `string`
+- `outputXML` → Type: `string`
+
+#### Right Panel — Output Structure
+- `xmlRecord` → Type: `Orders`
+- `Row[]`
+    - `index` (int)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)